src/components/Wizard/Wizard.tsx

```diff
@@ -119,61 +112,87 @@ export default function Wizard<T>({
 		resetAndClose();
 	}
 
+	function confirmTermination() {
+		setIsOpen(true);
+		confirm(props);
+	}
+
+	const props = {
```

Comment:
There is no reason to assign this const I'd argue. As it stands it deffo needs to be typed. If you pas the object inline you dodge hat need. The object is only used once..

Reply:
Moved to inline!